crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -268,6 +268,11 @@ impl EventTimelineItem {
         as_variant!(&self.kind, EventTimelineItemKind::Local(local) => &local.send_state)
     }
 
+    /// Get the local time that the event was enqueued at.
```

Comment:
"Local time" makes me think we're talking about time zones, and we're not:

```suggestion
    /// Get the time that the local event was pushed in the send queue at.
```